keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

```diff
@@ -526,302 +517,382 @@ dualstack cluster can have this flag configured as
    example, an IPv4 cluster can safely be upgraded to use
    `--service-cluster-ip-range=<ipv4-cidr>,<ipv6-cidr>`, but can not safely be
    changed to `--service-cluster-ip-range=<ipv6-cidr>,<ipv4-cidr>`.
-5. The first CIDR entry of `--service-cluster-ip-range=<first cidr entry>,
-   <secondary cidr entry>` is considered cluster default IP address family.
+5. The first CIDR entry of `--service-cluster-ip-range` is considered cluster
+   default IP address family.
```

Comment:
needs a "the" but also, should be "cluster default _service_ IP address family" or something, since it is not necessarily the default for other things. (eg, pod IPs could be in the reverse order)

(This would also require updates to later text if you change it here.)